puppet/modules/web/files/yum/HEADER.html

```diff
@@ -1,29 +1,26 @@
 <h1>yum.theforeman.org</h1>
 
-<h3>Stable Foreman releases</h3>
+<h3>Foreman</h3>
 
-<p>Releases are available under <code>/releases/VERSION/DIST/ARCH</code>, e.g.</p>
+<p>Foreman is available under <code>/releases/VERSION/DIST/ARCH</code>, e.g.</p>
 
 <ul>
-  <li>/releases/3.4/el8/x86_64</li>
+  <li>/releases/3.8/el8/x86_64</li>
   <li>/releases/nightly/el8/x86_64</li>
 </ul>
 
 <p>foreman-release RPMs containing an appropriate .repo file are available with fixed URLs:</p>
 
 <ul>
   <li><a href="https://yum.theforeman.org/releases/latest/el8/x86_64/foreman-release.rpm">https://yum.theforeman.org/releases/latest/el8/x86_64/foreman-release.rpm</a></li>
+  <li><a href="https://yum.theforeman.org/releases/nightly/el8/x86_64/foreman-release.rpm">https://yum.theforeman.org/releases/nightly/el8/x86_64/foreman-release.rpm</a></li>
 </ul>
 
 <p>Release packages <a href="https://theforeman.org/security.html#GPGkeys">are signed with a new key for each major release</a>.  The public key is available in the RPM-GPG-KEY-foreman file within each version directory or the foreman-release RPMs.</p>
 
-<p>A symlink is available at <code>/releases/latest</code> which always points to the latest stable release.  Please be careful when using this, as release upgrades often require some manual intervention (see release notes).</p>
-
-<h3>Nightly builds and release candidates</h3>
-
-<p>Nightly builds of Foreman are available under <code>/nightly/DIST/ARCH</code> and are refreshed a few times a day, but are not GPG signed.</p>
+<p>Nightly builds of Foreman are available under <code>/releases/nightly/DIST/ARCH</code> and are refreshed a few times a day, but are not GPG signed.</p>
 
-<p>Release candidates are in the versioned directories under <code>/releases</code>.  These are GPG-signed like stable packages.</p>
+<p>A symlink is available at <code>/releases/latest</code> which always points to the latest stable release.  Please be careful when using this, as release upgrades often require some manual intervention (see release notes).</p>
```

Comment:
can we drop this line (and the symlink)? :)

Reply:
Let's push that off to a separate discussion. I just wanted easy links to quickly install and verify https://github.com/theforeman/puppet-katello/pull/478 was no longer needed.